ui/src/components/TaskGroupStats/index.jsx

```diff
@@ -149,7 +149,7 @@ export default class TaskGroupStats extends Component {
       searchTerm,
       graphAll ? +Infinity : maxTasksInGraph
     );
-    const width = (sampledTasks.length + 1) * barWidth;
+    const width = (Math.max(25, sampledTasks.length) + 1) * barWidth;
```

Comment:
As whole graph is being stretched 100% we add here safe min `25` bars for it. So groups with just few bars, would not be stretched all the way, but would take those 25 placeholders instead